### PR TITLE
Bug 1813221: jsonnet/*: adapt resource requests

### PR DESCRIPTION
**Pull request by s-urbaniak**
This is a backport of https://github.com/openshift/cluster-monitoring-operator/pull/699

### CODE REVIEW

**Comment by lilic** [comment]
Your rebase went wrong I think.

**Comment by s-urbaniak** [comment]
should be good now, my local `origin/release-4.4` was out of date

**Comment by s-urbaniak** [comment]
prow is defunct it seems

File: jsonnet/jsonnetfile.lock.json [hunk 1]
```diff
@@ -1,7 +1,17 @@
 {
+  "version": 1,
   "dependencies": [
     {
-      "name": "etcd-mixin",
+      "source": {
```
Comment (lilic) [inline jsonnet/jsonnetfile.lock.json]: This is because of the v0.3.1 bump of jb, should we be introducing this to 4.4?

Comment (s-urbaniak) [reply jsonnet/jsonnetfile.lock.json]: ping @paulfantom

Comment (s-urbaniak) [reply jsonnet/jsonnetfile.lock.json]: fyi: the out of band consensus is that we use up-to-date jsonnet-bundler for backports and start pinning from the next sprint.

Comment (lilic) [reply jsonnet/jsonnetfile.lock.json]: Yes we agreed this is fine.

**Comment by lilic** [comment]
/retest

looks like things are still pending :D

**Comment by s-urbaniak** [comment]
I found we still have an issue with prometheus-operator and patch-merging containers. It doesn't work for `config-reloader`.

**Comment by s-urbaniak** [comment]
Leaving out `config-reloader` right now here, let's see if this passes e2e.

**Comment by lilic** [comment]
Can you just do a force push again seems like you hit a prow CI issue.

**Comment by s-urbaniak** [comment]
ack, i will also add the additional ones from https://github.com/openshift/cluster-monitoring-operator/pull/711

**Comment by lilic** [comment]
/hold

tests failed multuple times.

**Comment by paulfantom** [comment]
> tests failed multuple times.

I don't see failures, everything looks green.
/shrug

**Comment by lilic** [comment]
Weird 🤔 maybe was still cached the comment of the failures from the bot
/hold cancel

**Comment by s-urbaniak** [comment]
ping @bparees , can we get a cherry-pick approval?

**Pull request merged**